src/controllers/admin.controller.js

```diff
@@ -89,4 +77,22 @@ adminCtrl.deleteStory = async(req, res) => {
 
 };
 
+// AT-RESOURCES - Admin - Story list
+adminCtrl.renderStoryList = async (req, res) => {
+    let stories = [];
+    try{
+        const responseStoryList = await ssoServiceAPI.getAllStories();
+        if (responseStoryList === null || responseStoryList === undefined){
+            req.flash("error_msg", "Service unavailable");
+        } else {
+            console.log("--> adminCtrl.renderStoryList");
+            stories = responseStoryList.data;
+        }
+    } catch(err) {
+        console.error(err.message);
+    }  finally {
+        res.render("admin/story/list-story", { stories});
```

Comment:
it was res.render("admin/story/index", { stories});

Reply:
one question, in the direcory (story) exits the file index.hbs? or is rename the file list-story.hbs for index.hbs and render to this. Or is render to file index the directory admin.